specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json

```diff
@@ -813,6 +848,18 @@
         }
       }
     },
+    "ReservationBillingPlan": {
```

Comment:
>ReservationBillingPlan [](start = 5, length = 22)

nit - Until I saw the definition, it wasn't obvious that this is an enum. I wonder if it should be called `ReservationBillingPlanType` or similar.